api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -1081,6 +1081,8 @@ public static String getOpenmrsProperty(String property) {
 		
 		props
 				.add(new GlobalProperty(GP_DRUG_ORDER_DRUG_OTHER, "", "Specifies the uuid of the concept which represents drug other non coded"));
+		props.add(new GlobalProperty(LOGIN_URL, LOGIN_URL,
```

Comment:
Did you take a look at the expected parameters for the GlobalProperty constructor?

Reply:
yea it expects a property, a value, and a description

Reply:
Can you tell me the actual value that you are passing in for each of those?

Reply:
the value is the login.htm which is declared as a public by the LOGIN_URL

Reply:
I mean value for each of the parameters.
1) property, 2) value, 3) description

Reply:
1. login.htm, 2) login.htm, 3) "Responsible for defining the Authentication URL."

Mhh something seems wrong on the property and the value.

Reply:
That is exactly what i wanted you to see for yourself. 😊

Reply:
thanks.